content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -45,28 +45,28 @@ kind: Pod
 ...
 spec:
 	schedulerName: poseidon
```

Comment:
@furukawa3 , if you have time, would you clean up line 47 (tab)?

Reply:
+1 !!  I wanted to cleanup all of  *tab* space :)  However, my other patches were only removing trailing spaces and already merged...  (e.g. FR, ES docs)  So, is it OK to push as an another issue and patches to cleanup *tab* space ??

Reply:
Please lemme push an another patch to cleanup tab space if you feel it makes sense.